src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
 
-“Exactly! If they were listening; like Dark Lotus? “Juggalo Family?” That’s my real shit. That’s what I’m about.”
+I pushed myself up before the dudes could help me. I believed the shit about family, but [[tonight I could help myself|The Gutter - Outside hub]].]]]
 
-“It’s all about the money these days. Ninjas out there got on 4,000-dollar hatchet man jerseys and they think they the shit.”
+::The Gutter - Outside - Old-school juggalos 6
+“Aw shit,” Large raised his eyebrows, “sorry to hear that. Me and Kyle been bullshitting out here all night. Haven’t seen anything like that.”
 
-“Had a hole in my shirt the other day and I walk by one of them fancy-ass ninjas. You know what he said?”
+Kyle piped up. “Me and Ben’ll keep an eye out though for sure!”
 
-“Naw, man. No way! It ain’t about the clothes!”
+I thanked them and walked back towards the front of [[The Gutter|The Gutter - Outside hub]].
 
-“He called me a juggahoe. You heard that before? I laughed in his face. I told him it was all family, but I’d beat his ass if that was how he needed to be educated.”
+::The Gutter - Golf cart
+{(if: $cartPartsSeen is 4)[I figured I'd done enough damage to the cart, so I decided to keep looking around. But man, [[fuck Ron’s shitty cart|The Gutter - Outside hub]].]}
+\(if: $cartPartsSeen is <4)[Ron drove everywhere in this beaten-to-hell golf cart. I had no idea how he'd gotten it in the first place, especially since he couldn't hold down a job longer than six months. But, he loved to roll around the tarps showing it off every time he added another dumbass accessory to the thing.
 
-“Too true, man.”
+He had almost everything “tricked” out, including:
+(if: $wheelsSeen is false)[>>[[the wheels|The Gutter - Golf cart - Wheels]]]
+(if: $defenseSeen is false)[>>[[the defense system|The Gutter - Golf cart - Defense]]]
+(if: $hornSeen is false)[>>[[the awful horn|The Gutter - Golf cart - Horn]]]
+(if: $engineSeen is false)[>>[[the engine|The Gutter - Golf cart - Engine]]]]
 
-At this, they both took a moment to nod solemnly in unison, as if the two of them could hear a song no one else could.
+(set: $golfCartSeen to true)
 
-[[>>I didn’t wanna intrude, so I turned back to the sign’s light by the entrance|The Gutter - Outside]]
-(if: $iHatchet is 1)[>> [[I tried to casually walk past them so I could hear more|The Gutter - Outside - old-school juggalos - part 2]]]
+::The Gutter - Golf cart - Wheels
+Old clown-painted Barbie and Ken heads were glued around the rims of each wheel. The face painting was... oddly detailed.
 
-::The Gutter - Outside - old-school juggalos - part 2 <754,644>
-“Yo little ‘lette!” The larger one called out. “That’s a sick fucking hatchet you got. Come over here!”
+(if: $multitool is true)[I used the multitool to unscrew a few crucial wheel bolts. Just a (link: "little goof")[(set: $cartDamage to true)(go-to: "The Gutter - Golf cart")].](else:)[[[back|The Gutter - Golf cart]]]
```

Comment:
This was my original intention, but I pulled away from it while coding. We can revisit if we have more time, but the level of complexity you introduce when allowing the player to individually mess with the four items on the cart is pretty rough to handle, IMO. The original text I drafted for that idea didn't work as well as I thought, so we'd need code and text. Maybe someone else could do it faster, but I ditched it for this draft because I knew it would slow me way down.

FWIW, the options to mess with the cart only appear for folks with the multitool, so folks with the hatchet are truly just checking out the cart.

Reply:
I'll admit it felt kind of surprising to not have a choice here, since up to this point the player hasn't been forced to fuck with someone else's shit. (If anyone's got it coming, it's Ron, but also bear in mind the player _might not_ have met Ron yet.)

I think this is something we can address today when we get together to playtest. Changing the logic to give the player choice here doesn't seem too difficult, and I'd be happy to take it on. 

Related: either way, it might be good to precede these options with a little statement of mischievous intent from the player character. Something like: 

"Ron drove everywhere in this beaten-to-hell golf cart. I had no idea how he'd gotten it in the first place, especially since he couldn't hold down a job longer than six months. But, he loved to roll around the tarps showing it off every time he added another dumbass accessory to the thing.

So of course I couldn't resist fucking with it."